fsgs/FSGSDirectories.py

```diff
@@ -215,6 +306,7 @@ def get_screenshots_dir(cls):
 
     @classmethod
     @functools.lru_cache()
+    # FIXME: missing "Images" dir?
     def get_images_dir(cls):
```

Comment:
here we don't suggest an "Images" path liek in other methods, is there a reason?